src/chrome/content/rules/QBE.com.xml

```diff
@@ -0,0 +1,201 @@
+			- dplmdm.erp	(t)
+			- ebs.erp	(t)
+			- hyperion.erp	(t)
+			- obiee.erp	(t)
+			- hyperion.erp-ppd	(t)
+			- hyperion.erp-tst	(t)
+			- apro.erpppd	(t)
+			- dpl.erpppd	(t)
+			- dplmdm.erpppd	(t)
+			- ebs.erpppd	(t)
+			- obiee.erpppd	(t)
+			- dpl.erptst	(t)
+			- dplmdm.erptst	(t)
+			- ebs.erptst	(t)
+			- obiee.erptst	(t)
+			- lyncdiscover.es	(m)
+			- sip.es	(m)
+			- exchange	(t)
+			- lyncdiscover.fr	(m)
+			- sip.fr	(m)
+			- www.greenslip	(m)
+			- tst.greenslip	(r)
+			- group	(m)
+			- uat.group	(f)
+			- gsc-uat	(f)
+			- lyncdiscover.ie	(m)
+			- sip.ie	(m)
+			- lyncdiscover.it	(m)
+			- sip.it	(m)
+			- knowledgecentre	(t)
+			- lcportal	(t)
+			- mcarelearn	(t)
+			- miappsentryau	(r)
+			- misentry	(f)
+			- lyncdiscover.mx	(m)
+			- sip.mx	(m)
+			- ns1	(t)
+			- ns2	(t)
+			- quw.pega	(t)
+			- lyncdiscover.pr	(m)
+			- sip.pr	(m)
+			- uat	(f)
+			- uat	(t)
+			- uat.qeg	(f)
+			- dr.red	(i)
+			- lyncdiscover.se	(m)
+			- sip.se	(m)
+			- sentry	(ssl connection error)
+			- sentryau	(ssl connection error)
+			- sentrylmi	(t)
+			- staging	(m)
+			- dev.tpos	(f)
+			- eat.tpos	(f)
+			- sso.trn	(f)
+			- sso.uat	(f)
+			- enterpriseenrollment.uk	(m)
+			- idgate.uk	(m)
+			- lyncdiscover.uk	(m)
+			- sip.uk	(m)
+			- www.us	(m)
+			- enterpriseenrollment.us	(m)
+			- lyncdiscover.us	(m)
+			- sip.us	(m)
+			- webconf	(ssl connection error)
+			- webconf01	(ssl connection error)
+			- webmail	(ssl connection error)
+			- wifiaccess	(t)
+			- claim.workers	(t)
+			- trn.claim.workers	(t)
+			- uat.claim.workers	(t)
+
+		- (www.)qbeap.com	(m)
+		- www.qbelatam.com	(m)
+
+	e: expired certificate
+	f: ssl handshake failure
+	h: http redirect
+	i: invalid certificate chain
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="QBE.com">
+
+	<target host="qbe.com" />
+	<target host="www.qbe.com" />
+	<target host="access.qbe.com" />
+	<target host="access01.qbe.com" />
+	<target host="lyncdiscover.ao.qbe.com" />
+	<target host="lyncwebext01.ao.qbe.com" />
+	<target host="meet.ao.qbe.com" />
+	<target host="sip.ao.qbe.com" />
+	<target host="gsc.au.qbe.com" />
+	<target host="aunswctp.qbe.com" />
+	<target host="trn.aunswctp.qbe.com" />
+	<target host="autodiscover.qbe.com" />
+	<target host="ctpagentsonline.qbe.com" />
+	<target host="trn.ctpagentsonline.qbe.com" />
+	<target host="ctpfleet.qbe.com" />
+	<target host="trn.ctpfleet.qbe.com" />
+	<target host="dialin.qbe.com" />
+	<target host="easia.qbe.com" />
+	<target host="onevpn.eo.qbe.com" />
+	<target host="fastflow.qbe.com" />
+	<target host="greenslip.qbe.com" />
+	<target host="trn.greenslip.qbe.com" />
+	<target host="www.group.qbe.com" />
+	<target host="gsc.qbe.com" />
+	<target host="idgate-cust.qbe.com" />
+	<target host="lyncdiscover.qbe.com" />
+	<target host="lyncwebext01.qbe.com" />
+	<target host="meet.qbe.com" />
+	<target host="mivsp.qbe.com" />
+	<target host="Onevpn.na.qbe.com" />
```

Comment:
This target, `onevpn.na.qbe.com`, is listed twice

Reply:
Removed